lib/searchable_by.rb

```diff
@@ -43,8 +42,12 @@ def scope(&block)
     end
 
     def self.norm_values(query)
-      values = Shellwords.split(query.to_s)
-      values.flatten!
+      values = []
+      query  = query.to_s.dup
+      # rubocop:disable Style/PerlBackrefs
+      query.gsub!(/(\-|\+?)"([^"]*)"/) {|_| values.push("#{$1}#{$2}"); '' }
```

Comment:
Please see https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/PerlBackrefs, instead of disabling the rule, replace the $1 as suggested